generateData/csdbSchema.cql

```diff
@@ -0,0 +1,17 @@
+CREATE KEYSPACE IF NOT EXISTS airbnb
+WITH REPLICATION = {
```

Comment:
Does this line have to do with distributed systems?

Reply:
I'm not sure what you mean by distributed systems

Reply:
Yes, Cassandra has a "replication factor" that says how many "nodes" should have a copy of the data.